meta/main.yml

```
@@ -1,12 +1,15 @@
 # Standards: 1.8
 galaxy_info:
+  role_name: osbs_namespace
+  namespace: containerbuildsystem
   author: Luiz Carvalho
   description: Setup OpenShift namespace for OSBS usage
   company: Red Hat, Inc.
   license: BSD
   min_ansible_version: 2.1
   platforms:
-    name: EL
-    versions:
-    - 7
+    -
+      name: EL
+      versions:
+        - 7
```

Would this pass yamllint?
```suggestion
    - name: EL
      versions:
        - 7
```

---

Ah, it was saying "meta-no-info: Platforms should be a list of dictionaries", but I didn't **completely** understand what it wanted. Your suggestion is good.